cocaine/proxy/proxy.py

```diff
@@ -114,6 +116,49 @@
 
 X_COCAINE_HTTP_PROTO_VERSION = "X-Cocaine-HTTP-Proto-Version"
 
+class BodyProcessor(object):
+
+    def __init__(self, request, name, code, headers, init=''):
+        self.request = request
+        self.name = name
+        self.code = code
+        self.headers = headers
+        self.message = init
+
+    def __call__(self, part):
```

Comment:
I'd rather give it a meaningful name, like... well, `process`, instead of making objects callable.